workbench-core/environments/src/services/environmentService.test.ts

```diff
@@ -1038,7 +1040,8 @@ describe('EnvironmentService', () => {
         PROJ: projItem,
         ...env,
         provisionedProductId: '',
-        error: undefined
+        error: undefined,
+        datasetIds: undefined
```

Comment:
Why do we need `datasetIds: undefined`?

Reply:
Its a check to verify `datasetIds` does not appear in the final object (part of the AC)

Reply:
Would not having the line also verify that? If there is a `datasetIds` value, the `expect` would fail, right?

Reply:
`toEqual` only checks that there is field equality for the fields that exist. Not including a value for `datasetIds` would mean it wouldn't get checked. By defining it as `undefined`, we're explicitly saying that we want to make sure that `environment.datasetIds` is `undefined` on both objects.